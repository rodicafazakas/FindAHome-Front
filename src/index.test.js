import Index from "./index.js";

jest.mock("react-dom", () => ({ render: jest.fn() }));

describe("Application root", () => {
  it("renders without crashing", () => {
    expect(
      JSON.stringify(
        Object.assign({}, Index, { _reactInternalInstance: "censored" })
      )
    ).toMatchSnapshot();
  });
});
