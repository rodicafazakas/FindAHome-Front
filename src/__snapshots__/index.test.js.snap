// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application root renders without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
